app/root.tsx

```diff
@@ -98,58 +106,62 @@ function useChangeLanguage(locale: string) {
 }
 
 function App() {
-  return <Outlet />;
-}
-
-export function Layout({ children }: { children: ReactNode }) {
```

Comment:
Why did we get rid of the `<Layout />`? And then manually introduced it back to both the error and the app?